### PR TITLE
[#276] test smartcollect & linter fixes

### LINKED ISSUE #276: Smart collect
Test Beam brakes
Test has piece and piece tracking 
Smart collect

### PR DESCRIPTION
**Pull request by WindowsVistaisCool**
Closes #276

### CODE REVIEW

**Comment by MattD8957** [comment]
I'll look in 4th hour when I have my computer

**Comment by WindowsVistaisCool** [comment]
@MattD8957 when you do review dont merge just yet

File: src/main/java/frc/robot/command/shoot/SmartShoot.java
Comment (MattD8957) [inline src/main/java/frc/robot/command/shoot/SmartShoot.java:1]: Smart shoot doesnt have a java doc

File: src/main/java/frc/robot/subsystems/Climber.java
Comment (MattD8957) [inline src/main/java/frc/robot/subsystems/Climber.java:1]: Why remove set power and replace it with set setpoint when set set point already existed below?

Comment (WindowsVistaisCool) [reply src/main/java/frc/robot/subsystems/Climber.java:1]: I didn't, just moved overrides together

**Comment by MattD8957** [comment]
Please look at my comments
The linter was so close to perfect, just swerve.
The actual collect and index code is good 👍

**Review by MattD8957 — APPROVED** [review]
LGTM
have the checks passed they are just spinning circles here?

**Comment by WindowsVistaisCool** [comment]
> Please look at my comments The linter was so close to perfect, just swerve. The actual collect and index code is good 👍

Yeah I didn't touch swerve atm, but will do rn. Checks have passed BTW

**Comment by MattD8957** [comment]
Yeah as soon as I reviewed my screen updated

**Pull request merged**